lua/keymappings.lua

```diff
@@ -152,8 +153,22 @@ function M.config()
       ["<C-j>"] = { 'pumvisible() ? "\\<C-n>" : "\\<C-j>"', { expr = true, noremap = true } },
       ["<C-k>"] = { 'pumvisible() ? "\\<C-p>" : "\\<C-k>"', { expr = true, noremap = true } },
     },
+
+    operator_mode = {},
   }
 
+  -- Add some text objects
```

Comment:
Apart from being horribly difficult to read, what does this achieve ?
(Note that I didn't understand the PR's description)

Reply:
Just how we can select inside parentheses(`(, ), {, }, [, ]`), paragraphs(`p`), etc., with the code from this PR, we can now also select inside some special characters such as *, #, etc.